src/editor/tools/multi_select.h

```diff
@@ -74,6 +73,10 @@ struct MultiSelect {
 		return enabled_.size();
 	}
 
+        const std::set<int32_t>& getEnabled() const {
```

Comment:
Please use under_scores instead of CamelCase for function names:
```suggestion
        const std::set<int32_t>& get_enabled() const {
```